tutor_webhook_receiver/plugin.py

```diff
@@ -2,14 +2,45 @@
 import os
 import pkg_resources
 
+from .__about__ import __version__
 
 templates = pkg_resources.resource_filename(
     "tutor_webhook_receiver", "templates"
 )
 
-config = {}
+config = {
+    "add": {
+        "DB_PASSWORD": "{{ 8|random_string }}",
```

Comment:
Maybe make this longer, like 16 or 32 characters. 8 is silly short. :)

Reply:
That's right. Will do :)